prov/gni/src/gnix_nic.c

```diff
@@ -221,8 +222,11 @@ static int __nic_setup_irq_cq(struct gnix_nic *nic)
 	nic->irq_mmap_len = len;
 
 	if (nic->using_vmdh) {
-		info = _gnix_auth_key_lookup(GNIX_PROV_DEFAULT_AUTH_KEY,
-				GNIX_PROV_DEFAULT_AUTH_KEYLEN);
+		struct fi_gni_auth_key key;
```

Comment:
we are kind of old fashioned and like declarations to be at the beginning of the function